src/views/ModelPlan/ReadOnly/Table/index.tsx

```diff
@@ -53,6 +51,7 @@ const Table = ({
   const { t } = useTranslation('readOnlyModelPlan');
   const { t: h } = useTranslation('modelSummary');
   const { t: f } = useTranslation('home');
+  const { t: planBasicsT } = useTranslation('planBasics');
```

Comment:
I think this is the incorrect translation file, it should be

const { t: basicsT } = useTranslation('basics');

Reply:
Addressed in 6256b5d3c93dea3c07cc1ce0a457ef279bbc3f49